packages/app/src/cli/utilities/developer-platform-client/app-management-client.ts

```diff
@@ -357,8 +367,27 @@ export class AppManagementClient implements DeveloperPlatformClient {
     }
   }
 
-  async devStoresForOrg(_orgId: string): Promise<OrganizationStore[]> {
-    return []
+  // we are returning OrganizationStore type here because we want to keep types consistent btwn
+  // partners-client and app-management-client. Since we need transferDisabled and convertableToPartnerTest values
+  // from the Partners OrganizationStore schema, we will return this type for now
+  async devStoresForOrg(orgId: string): Promise<OrganizationStore[]> {
+    const storesResult = await businessPlatformOrganizationsRequest<ListDevStoresQuerySchema>(
+      ListDevStoresQuery,
+      this._businessPlatformToken ?? (await this.businessPlatformToken()),
+      orgId,
+    )
+
+    return storesResult.organization.properties.edges.map((edge) => {
+      const store = edge.node
+      return {
+        shopId: store.externalId,
+        link: store.primaryDomain,
+        shopDomain: store.primaryDomain,
+        shopName: store.name,
+        transferDisabled: true,
+        convertableToPartnerTest: true,
```

Comment:
Since you are doing this mapping in two places (here and on line 738) i'd extract this mapping to a function so we don't duplicate it. Something like `mapBPStoreToGenericStore` or probably a better name :P